### PR TITLE
Add management cmd for exporting audits to OTM2

### PR DESCRIPTION
**Pull request by steventlamb**
For standard models like Plot and Tree, records are exported using the standard django 'dumpdata' command, and imported using a custom fixture loader with a small DSL for field/value transformation rules.

OTM1 audits differ from OTM2 audits to an extent much greater than that of plots or trees. For this reason, we'll use a custom exporter to produce a fixture-like data dump that the OTM2 custom loader is able to understand.

### CODE REVIEW

File: treemap/management/commands/audit_export.py [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+                print "SKIPPING UNHANDLED NEW PHOTO: '{%s: %s}'" % (k, v)
+            elif k in model_rules.get('foreign_events', set()):
+                # TODO: figure out how to support these and add them
+                print "SKIPPING UNHANDLED FOREIGN EVENT: '{%s: %s}'" % (k, v)
+            else:
+                raise Exception("ERROR: UNHANDLED DIFF KEY - '{%s: %s}'" % (k, v))
+
+        return changes
+
+def get_audit_dicts(qs, model_name_l, model_name_u, audit_count):
```
Comment (jwalgran) [inline treemap/management/commands/audit_export.py]: It is not obvious what the `_l` and `_u` suffixes are here. From the calling code, it looks like they mean lower case and upper case. `model_name_lowercase` and `model_name_titlecase` are more self descriptive, but may make things too wordy. It's a toss up.

Comment (steventlamb) [inline treemap/management/commands/audit_export.py]: I'm glad you caught this. I meant to convert this to a single argument, `model_name`, and then convert to lower case in the function body.

File: treemap/management/commands/audit_export.py [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+        audit_count = 1
+        trees = Tree.objects.all()
+        plots = Plot.objects.all()
+
+        tree_hashes, tree_skipped, audit_count = get_audit_dicts(trees, 'tree', 'Tree', audit_count)
+        plot_hashes, plot_skipped, audit_count = get_audit_dicts(plots, 'plot', 'Plot', audit_count)
+
+        skipped = tree_skipped + plot_skipped
+
+        print "EXPORTED:", audit_count - 1, "audits"
```
Comment (jwalgran) [inline treemap/management/commands/audit_export.py]: Django says we should probably write these messages to stderr
https://docs.djangoproject.com/en/1.3/howto/custom-management-commands/

File: treemap/management/commands/audit_export.py [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+        print "SKIPPED:", skipped, "audits"
+        output = open(options['outfile'], 'w+b')
+        json.dump(tree_hashes + plot_hashes, output)
```
Comment (jwalgran) [inline treemap/management/commands/audit_export.py:215]: It it conventional for management commands to write to files directly or write to stdout. I prefer piping things around, but not at the expense of what is "normal" for Django.

**Comment by jwalgran** [comment]
+1
I saw you made a bunch of cards to finish this out. Are all the TODOs covered by cards? Since this code has a relatively short shelf life there is  a higher than average probability that TODOs won't happen.

File: treemap/management/commands/audit_export.py (hunk 3 above)
Comment (steventlamb) [inline treemap/management/commands/audit_export.py:215]: This is a tough one. We have two different types of output, the JSON payload, and the summary/warning text. Are you saying we should write the JSON payload to stdout and the summary text to stderr?

**Comment by ahinz** [comment]
+1 otm1 audits are sort of a nightmare
